python/README.md

```diff
@@ -23,7 +23,13 @@ ml4ir can be installed as a pip package by using the following command
 pip3 install ml4ir
 ```
 
-This will install **[ml4ir-0.0.2](https://pypi.org/project/ml4ir/)** (the current version) from PyPI.
+This will install **[ml4ir-0.1.3](https://pypi.org/project/ml4ir/)** (the current version) from PyPI.
```

Comment:
it would be cool if there was some kind of automated tool which took `${project.version}` and injected it into docs so they'd stay up to date.

Reply:
Yep, I will check for that. Actually, it might even be a better idea to remove the hardcoded version here.